metadata.json

```diff
@@ -19,7 +20,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.7.0 < 5.0.0"
+      "version_requirement": ">= 4.7.0 < 6.0.0"
```

Comment:
The hiera thing isn't even used.
I'm not sure if I should stick to 4.7 and use the params.pp or use 4.9 and properly use hiera.